include/seqan3/range/container/dynamic_bitset.hpp

```diff
@@ -1580,59 +1587,41 @@ class dynamic_bitset
      */
     //!\brief Performs element-wise comparison.
     template <size_t cap>
-    //!\cond
-        requires cap <= bit_capacity /* resolves ambiguousness when comparing two dynamic_bitsets of unequal capacity */
```

Comment:
(btw. you say the outcommenting std::ranges::iter_reference_t in line 1289 is not in the scope of this PR but here you also change something unrelated?)

Reply:
Yep, because this is self-contained, but this change would be needed across multiple files.

Reply:
If you want, I can change the requires back.

Reply:
no it's fine. just fix travis and I'll merge